### PR TITLE
Add section performance to Fusion docs

### PR DESCRIPTION
**Pull request by jordeu**
Signed-off-by: Jordi Deu-Pons <jordi@jordeu.net>

### CODE REVIEW

**Comment by pditommaso** [comment]
Nice

File: docs/fusion.rst [hunk 1]
```diff
@@ -171,6 +169,28 @@ Having the above configuration in place, you can run your pipeline using the fol
 
     nextflow run <YOUR PIPELINE> -work-dir s3://<YOUR BUCKET>/scratch
 
+
+NMVe storage
```
Comment (jordeu) [inline docs/fusion.rst:173]: It's NVMe

Comment (pditommaso) [reply docs/fusion.rst:173]: damn

Comment (pditommaso) [reply docs/fusion.rst:173]: Fixed 12782eeb

**Pull request closed without merging**